packages/amplify_datastore/android/build.gradle

```diff
@@ -57,8 +57,8 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "com.amplifyframework:aws-datastore:1.17.3"
-    implementation "com.amplifyframework:aws-api-appsync:1.17.3"
+    implementation "com.amplifyframework:aws-datastore:1.17.4"
+    implementation "com.amplifyframework:aws-api-appsync:1.17.4"
```

Comment:
I think we don't need this dependency on `aws-api-appsync` now that we have the API category correct? We should probably remove this in a separate PR as its not related to this change.

Reply:
Agree - we can take a look at that separately.